### PR TITLE
Feature/639 create new people_skill

### PR DESCRIPTION
**Pull request by kcinay055679**

### CODE REVIEW

File: app/views/people/people_skills/index.html.haml [hunk 1]
```diff
@@ -4,6 +4,7 @@
       %div.bg-skills-gray.d-flex.justify-content-center.people-skills-actions.me-2.mt-2.border.border-light-subtle.border-2
         = link_to image_tag("pencil-square.svg", {class: "me-1"})+ "Skills bearbeiten",
         people_skills_edit_person_path(@person), class: "d-flex align-items-center p-3"
+        = link_to "Skills ersellen", new_person_people_skill_path(@person), class: "d-flex align-items-center p-3", data: { turbo_frame: "remote_modal"}
```
Comment (kcinay055679) [inline app/views/people/people_skills/index.html.haml:7]: i18n

File: app/views/application/_remote_modal.html.haml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 = turbo_frame_tag "remote_modal" do
   .modal.fade{tabindex:"-1", "data-controller":"remote-modal", "data-action":"turbo:before-render@document->remote-modal#hideBeforeRender"}
+    %div#people-skill-wormhole
```
Comment (kcinay055679) [inline app/views/application/_remote_modal.html.haml:3]: rename

File: app/helpers/tab_helper.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module TabHelper
+  def person_tabs(person)
+    [
+      { title: 'CV', path: person_path(person) },
```
Comment (kcinay055679) [inline app/helpers/tab_helper.rb:6]: i18n

File: app/javascript/controllers/people_skills_controller.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            }
+        })
+    }
+
+    toggleLevel() {
+        this.labelTarget.textContent = this.levels[this.switchTarget.value - 1]
+    }
+
+    changeSkill(event) {
+        const skill_dropdown = event.target;
```
Comment (kcinay055679) [inline app/javascript/controllers/people_skills_controller.js:27]: prob refactor

File: app/javascript/controllers/people_skills_controller.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import { Controller } from "@hotwired/stimulus"
+import SlimSelect from 'slim-select'
+
+
+export default class extends Controller {
+    static targets = ['switch', 'label', 'skillsDD', 'categoriesDD', 'skillTitle']
```
Comment (Robin481) [inline app/javascript/controllers/people_skills_controller.js:6]: Naming... I have no Idea what `skillsDD` means

File: app/models/people_skill.rb [hunk 6]
```diff
@@ -16,6 +16,17 @@
 class PeopleSkill < ApplicationRecord
   belongs_to :person
   belongs_to :skill
+  accepts_nested_attributes_for :skill
+
+  def skill_attributes=(attributes)
```
Comment (Robin481) [inline app/models/people_skill.rb:21]: Don't put methods above all the validates, scopes etc.

**Review by Robin481 — CHANGES_REQUESTED** [review]
image

Two small comments, and make it look more aligned with the scroll to element pls.

**Pull request merged**